spring-cloud-alibaba-examples/integrated-example/docs/zh/kubernetes-deployment-zh.md

```diff
@@ -1,2 +1,121 @@
 # Spring Cloud Alibaba容器化部署最佳实践 | Kubernetes Helm-Chart 版本
-建设中
\ No newline at end of file
+
+## 准备工作
+
+此版本为 Spring Cloud Alibaba 最佳实践 Kubernetes 部署版本，其中需要您准备如下的环境。
+
+- Kubernetes（建议使用 Docker Desktop 内置集成的 Kubernetes 环境进行体验。）
+- Helm
+
+如果您还未具备如上的环境，请移步至对应官方文档进行环境搭建。
+
+- [Helm 安装](https://helm.sh/zh/docs/intro/install/)
+- [Kubernetes Docker Desktop 快捷安装](https://docs.docker.com/desktop/kubernetes/)
+
+此外，在启动测试前还需配置好 Kubernetes 主节点的 host 配置文件映射。
+
+```sh
+127.0.0.1 integrated-frontend
```

Comment:
You should list all NodePort name here, otherwise, we request the specific NodePort service that will failed. By the way, I found there isn't a NodePort name called `integrated-frontend`.